src/ui-setup/scripts/renderTypePageTest.js

```diff
@@ -32,183 +39,195 @@ const ${lowerName} = {
 ${renderPopulatedFields(fields, '  ')}
 }
 
-it('renders "${namePlural}" title', async () => {
-  const mocks = []
-  const { getByText } = await renderAndWait(<MockedProvider mocks={mocks} addTypename={false}>
-    <${namePlural}Page />
-  </MockedProvider>)
-
-  const title = getByText('${namePlural}')
-  expect(title).toBeInTheDocument()
-})
-
-it('renders a list of ${namePlural}', async () => {
-  const listMock = {
-    request: {
-      query: LIST_${capsNamePlural}_QUERY
-    },
-    result: {
-      data: {
-        list${namePlural}: ${lowerNamePlural}
-      }
-    }
-  }
-  const mocks = [listMock]
-  const { getAllByTestId } = await renderAndWait(<MockedProvider mocks={mocks} addTypename={false}>
-    <${namePlural}Page />
-  </MockedProvider>)
-
-  const ${lowerName}Cards = getAllByTestId('${lowerName}-card')
-  expect(${lowerName}Cards).toHaveLength(${lowerNamePlural}.length)
-})
-
-it('calls the create mutation', async () => {
-  const createMock = {
-    request: {
-      query: CREATE_${capsName}_MUTATION,
-      variables: { ${lowerName}Input: ${lowerName} }
-    },
-    newData: jest.fn(() => ({
-      data: {
-        create${name}: {
-          id: 1,
-          ...${lowerName}
+const testDescription = \`\${name} Entry Page\`
+
+const runUnitTest = () => {  
+  describe(testDescription, () => {
```

Comment:
remove line 42 (`const testDescription`) and this line should read
```
    describe(\`\${name} Entry Page\`, () => 
```

Reply:
reasoning: the final generated test files should just look like a normal test file.

Reply:
Did this for the integrated tests, but missed the unit ones. Already done. :+1: